qcut/docs/task/gmi-provider/seedance-2-0-260128/07-novel2video/02-shot-adapter.md

```
@@ -0,0 +1,169 @@
+	portraits: Record<string, string>, // character → HTTPS URL
+): AdaptedShot;
+```
+
+`portraits` is the already-uploaded HTTPS URLs keyed by character
+name. Subtask 3 (the handler) walks `portraits/registry.json`, runs
+each `front_view` through the subtask-1 uploader, and passes the
+resulting map into the adapter.
+
+## Variant selection
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)